### ISSUE TITLE
bag in new version

### ISSUE THREAD

**Issue by maystrenkoYurii** [postcss/postcss-custom-properties#186]
Bag in new version 9.0.1. With old version 8.0.10 good work
![Снимок](https://user-images.githubusercontent.com/23016311/59977774-2e164300-95de-11e9-9afe-4c6e5f69626c.PNG)
![Снимок](https://user-images.githubusercontent.com/23016311/59977797-6289ff00-95de-11e9-8322-5ecb99b0f492.PNG)

**Comment by montezume** [comment]
We are having the same issue 🙏

**Comment by jonathantneal** [comment]
I’m sorry I can’t help you right now. The details are far too sparse for me to setup a repeatable bug. This issue will be closed very soon if no additional information is provided.

That’s not what you want to hear. I would feel the same way, so let me explain. Without additional details, I want to close this issue in order to help you. See, if the open issue is not immediately visible to other affected users, those users may create new, similar issues with more details, or better yet code that lets me produce the issue. I request this in [CONTRIBUTING.md](https://github.com/postcss/postcss-custom-properties/blob/master/CONTRIBUTING.md#reporting-issues), which shows up for anyone opening issues.

However, if I leave this issue open, and if other affected users see this post — they may not take the time to check the details thoroughly. They’ll skim the issue enough to affirm it matches their own experience, and then add some kind of “me too”. I know this because I’ve done it, too. Unfortunately, that doesn’t help me, and it doesn’t help you, either. Actually, it’s kind of discouraging and distracting to know there are problems I want to fix but can’t. And it’s discouraging to you if I don’t help. I want to help you. I love helping you!

I know that it’s discouraging to you if I close this issue, too. I’m sorry about that.

Consider this — the best I can do with what you’ve given me is study the pictures, try to figure out how you are using PostCSS and PostCSS Custom Properties, and then copy the directories and file structure that I see in the pictures to try and recreate your circumstances. That’s tricky and time consuming, because most of the errors reference other projects. I don’t have the time to do all of that right now.

I really appreciate anyone who can setup a small project that reproduces this issue.

**Comment by montezume** [comment]
Sure, I can make a reproduction project in a little bit.

**Comment by jonathantneal** [comment]
Thank you so much!

**Comment by kevinansfield** [comment]
I'm having the same issue. Code in question:

```css
:root {
  --animation-speed-fast: 0.15s;
  --animation-speed-normal: 0.3s;
  --animation-speed-slow: 0.45s;
}
```

Error message:
```
postcss-custom-properties: <css input>:1:1: Unknown word
> 1 | 0.15s
    | ^
```

Unless this is invalid CSS it seems the issue may actually lie in the `postcss-values-parser` upgrade?

**Comment by kevinansfield** [comment]
Related upstream issue here https://github.com/shellscape/postcss-values-parser/issues/76

**Comment by kevinansfield** [comment]
[postcss-values-parser@3.0.5](https://github.com/shellscape/postcss-values-parser/releases/tag/v3.0.5) fixes the `s` value suffix bug so a dependency update would close this issue.

**Comment by jonathantneal** [comment]
This has been published in 9.0.2 🎉
